src/commands/math/basicSymbols.ts

```diff
@@ -487,30 +575,33 @@ optionProcessors.autoOperatorNames = function(cmds) {
   dict._maxLength = maxLength;
   return dict;
 };
-var OperatorName = P(Symbol, function(_, super_) {
-  _.init = function(fn) { this.ctrlSeq = fn; };
-  _.createLeftOf = function(cursor) {
+class OperatorName extends MQSymbol {
+  ctrlSeq:string;
+  constructor (fn?:string) {
+    super(fn || '');
```

Comment:
Is there any need to do this instead of giving a default value in the method signature?

Reply:
No. Those sound equivalent.